app/src/main/java/com/example/fizzbuzz/fizzbuzz.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.fizzbuzz
+
+fun main() {
+    fizzbuzz(100)
+}
+
+fun fizzbuzz(maxNumber: Int) {
+
+    for (i in 1..maxNumber) {
+        val sb = StringBuilder()
+        if (i % 3 == 0) sb.append("Fizz")
+        if (i % 5 == 0) sb.append("Buzz")
+        if(sb.isEmpty()) {
```

Comment:
Consider a more concise if statement here along the lines of `println(if (sb.isEmpty()) i else sb)`